src/opentime/rationalTime.cpp

```diff
@@ -118,7 +124,14 @@ RationalTime::from_timecode(
         unsigned int last_pos = 0;
         for (unsigned int i = 0; i < 4; i++)
         {
-            fields[i] = timecode.substr(last_pos, 2);
+            if (i < 3)
+            {
+                fields[i] = timecode.substr(last_pos, 2);
```

Comment:
hours can exceed 99? or is that disallowed? If we're going to catch high frame counts, probably we should think about the full parse, and do the split based on separators, not a two character heuristic, if 100+ hours is legal...

Reply:
I've purely based this on experimenting with 120 fps with Nuke and FFmpeg.

`ffmpeg -i <path> -timecode "00:00:00:119" <path>`

FFmpeg has no issue with that, and reports that timecode back.

Also Nuke reports it in the metadata and moving 1 frame forward it jumps to `00:00:01:00` and so on.

Reply:
can't help feeling that back-in-the-day they could have at least pre-empted over 100 fps and possibly over 99 hours. Doesn't seem unfathomable.

Reply:
To utilize framerates over 99, we'd have to modify the timecode parser and formatter to allow it ~ it's hardcoded for two digit codes at the moment.